Components/Profiling/Tracker/CMakeLists.txt

```diff
@@ -3,6 +3,10 @@ set(QUICC_CURRENT_SUBCOMPONENT_LIB Tracker)
 add_library(${QUICC_CURRENT_COMPONENT_LIB}_${QUICC_CURRENT_SUBCOMPONENT_LIB} STATIC
   Tracker.cpp
   MemoryTracker.cpp)
+
```

Comment:
Should we just set this in general and not for each component?

Reply:
the position independent code?
It is only needed for  the dynamic library that the jitter needs to call, so operators and everything used by them